File: core/src/com/mygdx/game/ship/NoInteraction.java
package com.mygdx.game.ship;

/**
 * Created by al on 5/17/2016.
 */
public class NoInteraction extends Interactable {
    public NoInteraction(){
        super(null);
        type = TileType.None;
    }

    @Override
    protected void addCoordinates() {

    }
}

File: web/src/ship/NoInteraction.ts
import Phaser from 'phaser';
import { Interactable } from './Interactable';
import { TileType } from './TileType';

/** Sentinel "no interactable here" value, mirroring NoInteraction.java. */
export class NoInteraction extends Interactable {
  constructor(scene: Phaser.Scene) {
    super(scene);
    this.type = TileType.None;
  }

  protected override addCoordinates(): void {}
}
